health-services/project-factory/src/server/utils/campaignUtils.ts

```
@@ -1695,21 +1697,37 @@ function mapBoundariesParent(boundaryResponse: any, request: any, parent: any) {
 
-      let totalTargetValue = 0;
-      for (const child of boundary.children) {
-        const childTargetValue = mapBoundary(child);
-        totalTargetValue += childTargetValue;
+    // Iterate through each child and accumulate their target values
+    for (const child of boundary.children) {
+      const childTargetValue = mapBoundary(child);
+
+      // Accumulate the child target values into the total target value
+      for (const key in childTargetValue) {
+        if (childTargetValue.hasOwnProperty(key)) {
```

Biome (1.9.4)

[error] Do not access Object.prototype method 'hasOwnProperty' from target object.

It's recommended using Object.hasOwn() instead of using Object.hasOwnProperty().
See MDN web docs for more details.

(lint/suspicious/noPrototypeBuiltins)